src/Private/Gui.ps1

```diff
@@ -49,10 +49,24 @@ function Start-PodeGuiRunspace
             [System.Reflection.Assembly]::LoadWithPartialName('PresentationFramework') | Out-Null
             [System.Reflection.Assembly]::LoadWithPartialName('PresentationCore') | Out-Null
 
-            # setup the WPF XAML for the server
-            $gui_browser = "
+            # Check for CefSharp
+            $loadCef = $false
+            if (
+                # This seems to be the safest method to detect if it is loaded or not, feel free to suggest a better one
+                ([AppDomain]::CurrentDomain.GetAssemblies() | Where-Object { $_.FullName.StartsWith("CefSharp.Wpf,") })
+            ) {
+                $loadCef = $true
+            }
```

Comment:
I'm wondering if this could be changed to just:

```powershell
$loadCef = [bool]([AppDomain]::CurrentDomain.GetAssemblies() | Where-Object { $_.FullName.StartsWith("CefSharp.Wpf,") })
```

🤔

Reply:
Yes, works fine, I've added it to the PR, thanks.